dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/event/JdbcEventStore.java

```diff
@@ -387,6 +387,7 @@ public List<ProgramStageInstance> getEvents( EventSearchParams params,
                     eventUser.setName( resultSet.getString( "user_assigned_name" ) );
                     eventUser.setFirstName( resultSet.getString( "user_assigned_first_name" ) );
                     eventUser.setSurname( resultSet.getString( "user_assigned_surname" ) );
+                    event.setAssignedUser( eventUser );
```

Comment:
Can you please add a test into TrackerEventsExportControllerTest 😄 The `JsonEvent.getAssignedUser` already exists for asserting 😋